bot.js

```diff
@@ -19,26 +24,69 @@ let fileLogger = (msg) => {
     });
 };
 
-let tweet = generate();
-while (tweet.length > TWEET_LENGTH) {
+function parseMessage(message) {
+	let tags = undefined;
+	let status = message;
+	let match = message.match(/\{\w+?\s+?.*?\}/g);
+	if (match) {
+		tags = {};
+		match.forEach(match => {
+			const tag = match.match(/\{(\w+)\s/)[1];
+			if (!tags[tag]) {
+				tags[tag] = match.match(/(\w+=".*?")/g).reduce((result, next) => { 
+					let key = next.match(/(\w+)=/)[1]; 
+					let value = next.match(/="(.*)"/)[1]; 
+					result[key] = value;
+					return result; 
+				}, {});
+			}
+			status = message.replace(match,'');
+		});	
+	}
+	return { 
+		status: status.trim().replace(/\s+/g,' '),
+		tags: tags
+	};
+}
+
+function postCardImageTweet(status, cardName) {
+	const outputfile = cardName.replace(/\s+/g, '-').toLowerCase() + '-' + uuid() + '.png';
+	const outputDir = '/tmp';
```

Comment:
I realized that this assumes the process has write permissions for /tmp . Is it safer to just write to the current directory or a local temp dir?